browser/src/core/Socket.js

```diff
@@ -1187,6 +1187,9 @@ app.definitions.Socket = L.Class.extend({
 				this._map.zotero._onMessage(subTextMsg);
 			}
 		}
+		else if (textMsg.startsWith('updateroutetoken') && window.indirectionUrl != '') {
```

Comment:
Looks good - do we have a wsd/protocol.txt update to reflect this ? would be nice to update the docs there for both the Admin and the client websocket.